platform/features/imagery/test/controllers/ImageryControllerSpec.js

```diff
@@ -174,6 +204,7 @@ define(
                         }
                     });
                     expect(unsubscribe).toHaveBeenCalled();
+                    expect(openmct.time.off).toHaveBeenCalled();
```

Comment:
Important to validate what it is called with here-- particularly the first argument.